docs/source/conf.py

```diff
@@ -10,14 +10,13 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
-import os
 import sys
 from distutils.sysconfig import get_python_lib
 from pathlib import Path
 
 import manim
 
-sys.path.insert(0, os.path.abspath("."))
+sys.path.insert(0, Path().resolve())
```

Comment:
> Hey there. I don't see why the tests are failing since I only changed `os.path` to pathlib functions. Maybe you can give some feedback in the review.

`sys.path` does not work with `Path`-like object (yet). The tests should pass when you convert it to string i.e. `str(Path().resolve())`.

While this was the solution back then when you made the pr, currently this line should be simply removed due to changed location of the `manim_directive`, making `sys.path` modification unnecessary.